commit 4cf42ec36673707acefcddbdc21da8d6a61ac5a2
Author: Lukas Bulwahn <lukas.bulwahn@gmail.com>
Date:   Wed Jul 20 13:37:17 2022 +0200

    nvmet-auth: select the intended CRYPTO_DH_RFC7919_GROUPS
    
    Commit 71ebe3842ebe ("nvmet-auth: Diffie-Hellman key exchange support")
    intends to select 'Support for RFC 7919 FFDHE group parameters' for using
    FFDHE groups for NVMe In-Band Authentication.
    
    It however selects CRYPTO_DH_GROUPS_RFC7919, instead of the intended
    CRYPTO_DH_RFC7919_GROUPS; notice the swapping of words here.
    
    Correct the select to the intended config option.
    
    Signed-off-by: Lukas Bulwahn <lukas.bulwahn@gmail.com>
    Reviewed-by: Hannes Reinecke <hare@suse.de>
    Signed-off-by: Christoph Hellwig <hch@lst.de>
    Signed-off-by: Jens Axboe <axboe@kernel.dk>

diff --git a/drivers/nvme/target/Kconfig b/drivers/nvme/target/Kconfig
index f0c91f7686a3..79fc64035ee3 100644
--- a/drivers/nvme/target/Kconfig
+++ b/drivers/nvme/target/Kconfig
@@ -93,7 +93,7 @@ config NVME_TARGET_AUTH
 	select CRYPTO_SHA256
 	select CRYPTO_SHA512
 	select CRYPTO_DH
-	select CRYPTO_DH_GROUPS_RFC7919
+	select CRYPTO_DH_RFC7919_GROUPS
 	help
 	  This enables support for NVMe over Fabrics In-band Authentication